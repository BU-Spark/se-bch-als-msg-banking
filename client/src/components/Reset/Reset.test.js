// This will be test for Reset.js
// Imports
import React from "react";
import Reset from "./Reset";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import 'regenerator-runtime/runtime'

// Tests:
// 1. Test that the Reset component renders

configure({adapter: new Adapter()});
test("Reset", () => {
    const wrapper = shallow(<Reset />);
    expect(wrapper).toMatchSnapshot();
}
);