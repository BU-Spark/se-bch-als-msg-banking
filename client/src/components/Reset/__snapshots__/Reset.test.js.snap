// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reset 1`] = `ShallowWrapper {}`;
